man/systemd-measure.xml

```diff
@@ -66,9 +66,9 @@
         <listitem><para>Pre-calculate the expected values seen in PCR register 11 after boot-up of a unified
         kernel image consisting of the components specified with <option>--linux=</option>,
         <option>--osrel=</option>, <option>--cmdline=</option>, <option>--initrd=</option>,
-        <option>--splash=</option>, <option>--dtb=</option>, <option>--pcrpkey=</option> see below. Only
-        <option>--linux=</option> is mandatory. (Alternatively, specify <option>--current</option> to use the
-        current values of PCR register 11 instead.)</para></listitem>
+        <option>--splash=</option>, <option>--dtb=</option>, <option>--pcrpkey=</option>,
+        <option>--tlinux=</option> see below. Only <option>--linux=</option> is mandatory. (Alternatively,
```

Comment:
hmm, i find the concept of calling this "trusted" a bit weird, because as long as we deal it we now trust it *less* than before, given we ask the firmware to authenticate it *again*.

I think "signed" is the right word, as you had before, because that#s what it is: a signed kernel as opposed to an unsigned one as before.

could also call it `.sblinux`, to indicate it comes with SecureBoot signature?